index.js

```diff
@@ -2,7 +2,7 @@
 /* eslint import/no-commonjs: 0 */
 
 // eslint-disable-next-line no-process-env
-if (process.env.ZOID_FRAME_ONLY) {
+if (process && process.env && process.env.ZOID_FRAME_ONLY) {
```

Comment:
this will still throw error since process is not defined in browser.

``` js
if (typeof process !== 'undefined' && process.env && process.env.ZOID_FRAME_ONLY) {
```